README.md

```diff
@@ -115,15 +86,31 @@ useful if you have several versions installed:
 
 Note that Regula requires Terraform 0.12+ in order to generate the JSON-formatted plan.
 
-### Windows
+#### Windows
 
 Because Regula uses a bash script to automatically generate a plan, convert it to JSON, and run the Rego validations, Windows users can instead manually run the steps that Regula performs. See those steps [here](#locally-producing-a-report-on-windows).  Alternatively, you can run the script using [WSL](https://docs.microsoft.com/en-us/windows/wsl/about).
 
+### Running Regula with Docker
+
+Regula is available as a Docker image on DockerHub [here](https://hub.docker.com/r/fugue/regula).
+
+To run Regula on a CloudFormation template or Terraform plan file, use the following command:
+
+    docker run -i fugue/regula [IAC_TEMPLATE]
```

Comment:
I think we need a shell redirect in there:
```suggestion
    docker run -i fugue/regula < [IAC_TEMPLATE]
```
I usually like to include `--rm` for temporary containers too
```suggestion
    docker run --rm -i fugue/regula < [IAC_TEMPLATE]
```